Array/ArrayHomework1.php

```diff
@@ -0,0 +1,30 @@
+<?php
+$calculate = array(
+    array(5, 12, 17, 9, 3),
+    array(13, 4, 8, 14, 1),
+    array(9, 5, 3, 7, 21)
+);
```

Comment:
You misunderstood the homework. The example Inputs/Output are separated 1-dimensional array, Please check hint at https://gitlab.com/pascaliaasia/pav-training/-/blob/basic-training/data-structure/homework-hint/array-hint.md